googleapiclient/errors.py

```diff
@@ -44,9 +44,9 @@ def _get_reason(self):
     """Calculate the reason for the error from the response content."""
     reason = self.resp.reason
     try:
-      data = json.loads(self.content)
+      data = json.loads(self.content.decode('utf-8'))
```

Comment:
I think that this should not be included within this block, it should be in a separate one before this.

If there is actually a 'reason' but it fails on UnicodeError, it will seem as if there is no reason when in fact there is.